tracelib/trace.js

```diff
@@ -43,16 +43,17 @@ function parseInfo(info) {
   return info;
 }
 
-module.exports.options = options;
-var enabled = Boolean(options.traceFile);
+let module = {exports: {}};
+export default module.exports;
+module.exports.enabled = false;
+module.exports.enable = function() {
+  module.exports.enabled = true;
+  init();
+};
+//module.exports.options = options;
```

Comment:
Intentional?

Reply:
Added a TODO to add back support for options, we haven't needed this lately so I didn't think it was worth adding a dependency for just this.